app/notification.go

```diff
@@ -658,7 +658,7 @@ func (m *ExplicitMentions) addGroupMention(word string, groups map[string]*model
 		m.GroupMentions = make(map[string]*model.Group)
 	}
 
-	m.GroupMentions[group.Name] = group
+	m.GroupMentions[*group.Name] = group
```

Comment:
Wouldn't this—and any other instances—require a `nil` check to avoid a nil pointer dereference?

Reply:
I had convinced myself it wasn't necessary. But let me double check.

Reply:
Agree.  I wrapped these in a check.